src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,92 @@
-export const Pagination = () => {};
+import React from 'react';
+
+type Props = {
+  total: string[];
+  perPage: number;
+  currentPage: number;
+  onPageChange?: (value: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagination = Math.ceil(total.length / perPage);
+
+  function onClickHandler(
+    e: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    chek: number,
+  ) {
+    e.preventDefault();
+    if (chek === 1) {
+      if (currentPage > 1 && onPageChange) {
+        onPageChange(currentPage - 1);
+      }
+    }
+
+    if (chek === 2) {
+      if (currentPage < pagination && onPageChange) {
+        onPageChange(currentPage + 1);
+      }
+    }
+  }
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            onClick={e => onClickHandler(e, 1)}
```

Comment:
better to create separate function for prev and next, instead of one complex with strange checks